packages/node_modules/@webex/internal-plugin-devices/test/unit/spec/devices.js

```diff
@@ -1,7 +1,51 @@
 import {assert} from '@webex/test-helper-chai';
+import {cloneDeep} from 'lodash';
+import Devices from '@webex/internal-plugin-devices';
+import MockWebex from '@webex/test-helper-mock-webex';
+
+import dto from './wdm-dto';
 
 describe('plugin-devices', () => {
-  beforeEach('initialize webex with the devices plugin', () => {});
+  let webex;
+  let devices;
+
+  beforeEach('initialize webex with the devices plugin', () => {
+    webex = new MockWebex({
+      children: {
+        devices: Devices
+      }
+    });
+
+    const deviceConfig = cloneDeep(dto);
+
+    webex.internal.devices.set(deviceConfig);
+
+    devices = webex.internal.devices;
+  });
+
+  describe('#derived', () => {
```

Comment:
Since this isn't really a function, let's just write it out
```suggestion
  describe('derived properties', () => {
```

Reply:
Change has been pushed.